lib/danger/comment_generators/github.md.erb

```diff
@@ -22,8 +22,8 @@
     :book: | <%= message %><% end %>
 <% end %>
 
-<p align="right">
-  Generated by :no_entry_sign: danger
+<p align="right" meta="generated_by_danger">
```

Comment:
I think it's pretty reasonable that people won't be putting that in their comments, but this can change to something harder to reproduce if we want, there's tests for it now.